### PR TITLE
Flip staging to ES7

### PR DESCRIPTION
**Pull request by sravfeyn**
@millerdev

### CODE REVIEW

**Comment by sravfeyn** [comment]
This is already deployed

**Comment by sravfeyn** [comment]
Please hold on merging. I had to roll this back

File: environments/staging/inventory.ini.j2 [hunk 1]
```diff
@@ -105,10 +105,12 @@ redis2
 
 [elasticsearch:children]
 # Amazon EC2
-es4
-es5
-#es6 this is intentional until testing is over, please contact sravan before modifying these
-#es7
+# old ES v2 hosts
+# es4
+# es5
```
Comment (millerdev) [inline environments/staging/inventory.ini.j2:110]: Can these lines be deleted (rather than commented)?

Comment (sravfeyn) [reply environments/staging/inventory.ini.j2:110]: We can delete them once QA is pass.

**Pull request merged**